### PR TITLE
Added support to upload keyfile for BigQuery

### PR DESCRIPTION
**Pull request by vlbe**

### CODE REVIEW

File: src/modules/CdmSourceList/components/List/ModalCreateEdit/container.js [hunk 1]
```diff
@@ -59,11 +59,28 @@ ModalCreateEdit.propTypes = {
...
 };
 
+function validateForm(state) {
+  const formValues = getFormValues(form.createDataSource)(state);
+  const dbmsType = get(state, 'form.createDataSource.values.dbmsType');
+  let requiredFields = ['name', 'dbmsType', 'connectionString', 'cdmSchema'];
+  if (dbmsType === 'BIGQUERY') {
+    const dataSourceData = get(state, 'cdmSourceList.dataSource.queryResult.result', {}, 'Object');
+    const isEdit = dataSourceData && dataSourceData.id;
+    if (!isEdit) {
```
Comment (pavgra) [inline src/modules/CdmSourceList/components/List/ModalCreateEdit/container.js]: What is the logic related to the `if`?

File: src/modules/CdmSourceList/components/List/ModalCreateEdit/style.scss [hunk 2]
```diff
@@ -50,6 +50,10 @@
 				display: flex;
 				flex-direction: column;
 				padding: 0 1.5rem 1.5rem 1.5rem;
+
+				.ac-button:disabled {
+					background-color: #E1E1E1;
```
Comment (pavgra) [inline src/modules/CdmSourceList/components/List/ModalCreateEdit/style.scss]: Don't we have a matching variable?

File: src/modules/CdmSourceList/components/List/ModalCreateEdit/container.js (hunk 1 above)
Comment (vlbe) [reply src/modules/CdmSourceList/components/List/ModalCreateEdit/container.js]: if the action is not editing existing data source, the `keyfile` is required. While editing user can upload new keyfile to overwrite existing one.

File: src/modules/CdmSourceList/components/List/ModalCreateEdit/style.scss (hunk 2 above)
Comment (vlbe) [reply src/modules/CdmSourceList/components/List/ModalCreateEdit/style.scss]: fixed

**Comment by vlbe** [comment]
Merged in https://github.com/OHDSI/ArachneUI/pull/701

**Pull request closed without merging**